src/menu/MenuItem.ts

```diff
@@ -0,0 +1,118 @@
+import { assign } from '@dojo/core/lang';
+import { HNode } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+import ThemeableMixin, { theme, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import * as css from './styles/menu.css';
+
+export interface MenuItemProperties extends ThemeableProperties {
+	/**
+	 * Indicates whether the menu is disabled. If true, then the widget will ignore events.
+	 */
+	disabled?: boolean;
+
+	/**
+	 * Applies only when a URL is provided. If `true`, the URL will be opened in a new window.
+	 */
+	external?: boolean;
```

Comment:
Wouldn't a user just put an anchor as the child to this MenuItem? I don't think the MenuItem should handle link behavior.